Traceback (most recent call last):
  File "D:\CVPROJECTS\FlowIDE\main.py", line 180, in <module>
    wrapper(main)
  File "C:\Users\timid\AppData\Local\Programs\Python\Python310\lib\curses\__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
  File "D:\CVPROJECTS\FlowIDE\main.py", line 171, in main
    stdscr.move(y,x)
_curses.error: wmove() returned ERR
